index.html

```diff
@@ -132,5 +123,251 @@
     
         </section>
     </header>
+
+    <main class="works">
+        <div class="card-works">
+                <div class="work-snapshot">
+                </div>
+                <div class="work-block">
+                    <div class="primary-text">
+                        <h1 class="project-title">Tonic</h1>
+                        <div class="frame">
+                            <div class="client"><p>CANOPY</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="role"><p>Back&nbspEnd&nbspDev</p></div>
+                            <div>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </div>
+                            <div class="year"><p>2015</p></div>
+                        </div>
+                    </div>
+                    <h2 class="project-details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
```

Comment:
- [ Required ] Please consider changing the project descriptions in your `works` section to a `p` tag, the best tag to use when marking up multi-line text is the `p` tag. The `hx` - `h1` - `h2`  tags are usually used to markup headings on a webpage and this is not a heading :smile: